tests/util/test_object.py

```diff
@@ -1,7 +1,47 @@
 ''' Tests for datamegh.util.object util. '''
 
 import pytest
-from datamegh.util.object import linearize, delinearize, merge
+from datamegh.util.object import linearize, delinearize, merge, dict_to_list
+
+
+def test_dict_to_list_1():
+    '''
+    Test a dictionary converted to list.
+    '''
+    value = {
+        'foo': 'bar',
+        'just': 'test',
+        'hello': 'world',
+    }
+
+    expected = [
+        {'name': 'foo', 'value': 'bar'},
+        {'name': 'just', 'value': 'test'},
+        {'name': 'hello', 'value': 'world'}
+    ]
+
+    assert dict_to_list(value) == expected
+
```

Comment:
PEP8 requires two spaces in between function declarations. 

```suggestion


```

Reply:
Updated

Reply:
Will BLACK be doing this later? @kabirbaidhya

Reply:
Yes but black follows PEP8 too.